src/pages/HomePage.tsx

```diff
@@ -0,0 +1,8 @@
+import { FC } from 'react';
+import DepartmentsList from '../components/DepartmentsList';
+
+const HomePage: FC = () => {
+  return <DepartmentsList />;
```

Comment:
doesn't look like DepartmentsList belongs to HomePage. 
Much better if you make DepartmentsList a default router. But common app parts should be moved outside of router

Reply:
@DariaRusovich this one is unresolved. Either remove the file either resolve :)